src/scenes/UserGalleryPage/UserGalleryCollection.tsx

```diff
@@ -129,37 +129,40 @@ function UserGalleryCollection({ queryRef, collectionRef, mobileLayout }: Props)
           <StyledCollectorsTitle onClick={handleViewCollectionClick}>
             {unescapedCollectionName}
           </StyledCollectorsTitle>
-          <StyledSettingsDropdown>
+          <StyledOptionsContainer>
             {isHovering && (
-              <Dropdown>
-                {showEditActions && (
-                  <>
-                    <TextButton
-                      onClick={handleEditNameClick}
-                      text="EDIT NAME & DESCRIPTION"
-                      underlineOnHover
-                    />
-                    <Spacer height={8} />
-                    <TextButton
-                      text="Edit Collection"
-                      onClick={handleEditCollectionClick}
-                      underlineOnHover
-                    />
-                    <Spacer height={8} />
-                  </>
-                )}
-                <TextButton
-                  text="View Collection"
-                  onClick={handleViewCollectionClick}
-                  underlineOnHover
-                />
-                <Spacer height={8} />
-                <CopyToClipboard textToCopy={collectionUrl}>
-                  <TextButton text="Share" onClick={handleShareClick} />
-                </CopyToClipboard>
-              </Dropdown>
+              <StyledCopyToClipboard textToCopy={collectionUrl}>
+                <StyledShareButton text="Share" onClick={handleShareClick} />
+              </StyledCopyToClipboard>
             )}
-          </StyledSettingsDropdown>
+            <StyledSettingsDropdown>
```

Comment:
nit: the "SHARE" text and '...' dropdown menu button is bit vertically off center
![Screen Shot 2022-06-08 at 22 24 23](https://user-images.githubusercontent.com/80802871/172632159-47a4fceb-1f17-4b38-88dc-96d7ab9151c9.png)


spec:
![Screen Shot 2022-06-08 at 22 24 38](https://user-images.githubusercontent.com/80802871/172632189-209ade93-70c2-4e84-893f-7d3d23138b2f.png)

Reply:
Fixed using `background-position: center`